torch/csrc/Module.cpp

```diff
@@ -466,6 +466,20 @@ PyObject *THPModule_benchmarkCuDNN(PyObject *_unused, PyObject *noargs)
   else Py_RETURN_FALSE;
 }
 
+PyObject *THPModule_setUseTF32CuBLAS(PyObject *_unused, PyObject *arg)
+{
+  THPUtils_assert(PyBool_Check(arg), "set_use_tf32_cublas expects a bool, "
+          "but got %s", THPUtils_typename(arg));
+  at::globalContext().setUseTF32CuBLAS(arg == Py_True);
+  Py_RETURN_NONE;
+}
+
+PyObject *THPModule_useTF32CuBLAS(PyObject *_unused, PyObject *noargs)
+{
+  if (at::globalContext().useTF32CuBLAS()) Py_RETURN_TRUE;
+  else Py_RETURN_FALSE;
```

Comment:
nit: `Py_RETURN_TRUE` and `Py_RETURN_FALSE` are macros so i feel nervous using them in an if/else without braces.  If cpython changes so they aren't one liners, the if/else will break.

I guess many other functions in this file use similar braceless conditionals and would also break, so keep it if you want.